app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,162 @@
+                  <li class="card_li"><strong><a href="https://uottawa.libguides.com/DH/textual">Digital Humanities Guide at University of Ottawa</a></strong>: This is a great guide that highlights several approaches to text analysis in the field.</li>
+                  <li class="card_li"><strong><a href="http://voyant-tools.org/">Voyant Tools</a></strong>: This is more of a tool than an introductory resource, but it's a great "gateway drug" into the powers of text analyis.</li>
+                </ul>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <div class="auk_deriv_accord card">
+    <div class="auk_deriv_accord-header card-header" id="headingThree">
+      <h5 class="mb-0">
+        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">Domains File
+        </button>
+      </h5>
+    </div>
+    <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion">
+      <div class="auk_deriv_accord_body card-body">
+        <p class="about_p">Sometimes you might just want to know what is inside a web archive! The web is a complex beast: a small web archive that aims to collect just one page might end up capturing embedded widgets and content from other websites just to make something play. Alternatively, a crawl might have been corrupted and begun to find material that's far out of scope.</p>
+        <table id="auk_deriv_accord_table">
+          <tbody class="auk_deriv_accord_table">
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>Tutorial</strong></td>
+              <td class="auk_deriv_td_table"><%=link_to("Making Sense of the Domains Count", "/derivatives/domains")%></td>
+            </tr>
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>File Output</strong></td>
+              <td class="auk_deriv_td_table"><kbd>.txt</kbd></td>
+            </tr>
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>Files Generated</strong></td>
+              <td class="auk_deriv_td_table">A <strong>text file</strong> containing the frequency count of domains captured within your web archive.</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <br />
+  <h3 class="about_h3">How to Cite Archives Unleashed</h3>
+  <p class="about_p">If you've found Archives Unleashed Toolkit or Cloud helpful in your research, please consider using the citation below in your publications. Your citations help to futher the recognition of using open-source tools for scientific inquiry, assists in growing the web archiving community, and acknowledges the efforts of contributors to this project.
```

Comment:
minor (possible) typo fix - let's add "the" before "Archives Unleashed Toolkit"

(it could go either way but we generally seem to talk about "the" toolkit)

also typo fix - "futher" -> "further"